charts/linkerd2/templates/prometheus.yaml

```diff
@@ -274,3 +274,25 @@ spec:
       - {{- include "partials.proxy.volumes.labels" . | indent 8 | trimPrefix (repeat 7 " ") }}
       {{ end -}}
       - {{- include "partials.proxy.volumes.identity" . | indent 8 | trimPrefix (repeat 7 " ") }}
+---
+{{- if and .Values.prometheusPersistence.enabled }}
```

Comment:
I think if you move this if up a line to above the `---` then you won't get the spurious extra `---` separator when this is disabled.

Reply:
Good point, thanks!